app/controllers/members_controller.rb

```diff
@@ -49,4 +55,10 @@ def check_admin_rights
         redirect_to group_members_path(@group), alert: "You don't have the rights to do that."
       end
     end
+
+    def add_group_user(user)
+      return 'User not found.' unless user
```

Comment:
Used to be like this, and I changed it. Oops. I think I will put it back if you think it's less confusing. I kinda find it easier to read the flow than with nested `if...else` but I get your point.